bin/run.sh

```diff
@@ -32,4 +18,143 @@ if [[ ! -v OSTORLAB_PLATFORM ]]; then
   exit 1
 fi
 
-exec java -jar ${PLUGIN_JAR} --api-key $OSTORLAB_API_KEY --file-path $OSTORLAB_FILE_PATH --artifacts-dir $OSTORLAB_ARTIFACTS_DIR --platform $OSTORLAB_PLATFORM
+# Add --scan-profile argument
+PLATFORM=""
+if [[ "${OSTORLAB_PLATFORM,,}" == "android" ]]; then
+  PLATFORM="android-apk"
+elif [[ "${OSTORLAB_PLATFORM,,}" == "ios" ]]; then
+  PLATFORM="ios-ipa"
+elif [[ "${OSTORLAB_PLATFORM,,}" == "link" ]]; then
+  PLATFORM="link"
+else
+  echo "Unsupported platform: ${OSTORLAB_PLATFORM}"
+  exit 1
+fi
+
+
+# Add optional --title argument if OSTORLAB_TITLE is set
+TITLE_ARG=""
+if [[ -n "${OSTORLAB_TITLE}" ]]; then
+  TITLE_ARG="--title=${OSTORLAB_TITLE}"
+else
+  TITLE_ARG="--title=Gitlab scan"
+fi
+
+# Add optional --break-on-risk-rating argument if OSTORLAB_RISK_THRESHOLD is set
+BREAK_ON_RISK_RATING=""
+if [[ -n "${OSTORLAB_RISK_THRESHOLD}" ]]; then
+  BREAK_ON_RISK_RATING="--break-on-risk-rating=${OSTORLAB_RISK_THRESHOLD}"
+fi
+
+# Add optional --max-wait-minutes argument if OSTORLAB_MAX_WAIT_MINUTES is set
+MAX_WAIT_MINUTES=""
+if [[ -n "${OSTORLAB_MAX_WAIT_MINUTES}" ]]; then
+  MAX_WAIT_MINUTES="--max-wait-minutes=${OSTORLAB_MAX_WAIT_MINUTES}"
+fi
+
+# Add --scan-profile argument
+SCAN_PROFILE=""
+if [[ "${OSTORLAB_SCAN_PROFILE}" == "Full Scan" ]]; then
+  SCAN_PROFILE="--scan-profile=full_scan"
+elif [[ "${OSTORLAB_SCAN_PROFILE}" == "Full Web Scan" ]]; then
+  SCAN_PROFILE="--scan-profile=full_web_scan"
+else
+  SCAN_PROFILE="--scan-profile=fast_scan"
+fi
+
+# Add URLS as arguments if OSTORLAB_URLS is set
+URLS=""
+if [[ -n "${OSTORLAB_URLS}" ]]; then
+  for url in ${OSTORLAB_URLS}; do
+    URLS+="--url=${url} "
```

Comment:
The  `--method` argument required.. 
```suggestion
    URLS+="--url=${url} --method GET"
```

Reply:
The method parameter does not exist in the API.